test/runtests.jl

```diff
@@ -35,51 +36,43 @@ function gvf_tests()
     @testset "GVF Tests" begin
         @testset "GVF" begin
             @test test_construction(GVF, FeatureCumulant(1), ConstantDiscount(0.9), NullPolicy())
-        end
-
-        @testset "PredictionGVF" begin
-            # GVF builds
-            @test test_construction(PredictionGVF, FeatureCumulant(1), ConstantDiscount(0.9), NullPolicy())
-            @test test_construction(PredictionGVF, FeatureCumulant(1), ConstantDiscount(0.9))
 
-            # GVF returns the right thing
-            gvf = PredictionGVF(FeatureCumulant(2), ConstantDiscount(0.78), NullPolicy())
-            @test all(get(gvf, [1,2,3]) .== [2, 0.78, 1.0])
+            gvf = GVF(FeatureCumulant(2), ConstantDiscount(0.78), NullPolicy())
+            @test all(get(gvf, [5,5,5], 1, [1,2,3]) .== [2, 0.78, 1.0])
+            @test all(get(gvf, [5,5,5], 1, [1,2,3], [0.1, 0.2, 0.3]) .== [2, 0.78, 1.0])
             @test all(get(gvf, [5,5,5], 1, [1,2,3], 2, [0.1,0.2,0.3]) .== [2, 0.78, 1.0])
         end
+        # Previous implementation.
```

Comment:
Looks like a hanging comment. Probably good without historically referencing comments anyways, after a few months we'll forget why this comment was here in the first place :)